floss/render/default.py

```diff
@@ -134,20 +135,22 @@ def render_staticstrings(strings, ostream, verbose, disable_headers):
         unicode_offset_len = len(f"{unicode_strings[-1].offset}")
     offset_len = max(ascii_offset_len, unicode_offset_len)
 
-    render_sub_heading("FLOSS STATIC STRINGS: ASCII", len(ascii_strings), ostream, disable_headers)
+    render_sub_heading("FLOSS STATIC STRINGS: " + colored("ASCII", "blue", attrs=['bold']), len(ascii_strings), ostream, disable_headers)
```

Comment:
the implementation here is good, and i think it can be even further polished by refactoring the common coloring configs into their own functions, like `def heading_style(str) -> str` and `def string_style(str) -> str`. the benefits are twofold:

  1. a bunch of lines are simplified, and
  2. if the config changes, like changing the color, its easy to do in one place, versus many.

if you agree, would you please update the PR with these changes?